package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build && next export",
+    "postbuild": "next-sitemap",
```

Comment:
Just adding this doesn't seem to be enough. 
I think the process for building should now be:
```sh
# generate stuff once
yarn build
# generate new static files
yarn postbuild
# generate the entire site again -- copying the generated files
yarn build
```

Or in one line:
`"build": "next build && yarn postbuild && next build && next export"`

Alternatively I guess one could put the generated files directly into "out", and explicitly call "postbuild" during deploy.

Reply:
I know this is a cliché, but it works on my machine. When I run yarn build, it also runs postbuild right after build

Reply:
Nvm I was wrong. Changes made.